src/main/java/uk/gov/companieshouse/chdorderconsumer/ChdOrderConsumerApplication.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.companieshouse.chdorderconsumer;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class ChdOrderConsumerApplication {
+
+	public static final String APPLICATION_NAMESPACE = "chd-order-consumer";
+
+	public static void main(String[] args) {
+		SpringApplication.run(ChdOrderConsumerApplication.class, args);
+	}
+
+}
```

Comment:
This should have spaces instead of tabs.
Also, the APPLICATION_NAMESPACE should be in a LoggingUtils class (see certified-copies.orders.api.ch.gov.uk as an example)

Reply:
done